packages/react-core/src/components/HelperText/HelperTextItem.tsx

```diff
@@ -70,7 +70,11 @@ export const HelperTextItem: React.FunctionComponent<HelperTextItemProps> = ({
           {variant === 'error' && <ExclamationCircleIcon />}
         </span>
       )}
-      <span className={css(styles.helperTextItemText)}>{children}</span>
+
+      <span className={css(styles.helperTextItemText)}>
+        {children}
+        {isDynamic && <span className="pf-u-screen-reader">: {variant} status;</span>}
```

Comment:
I don't think you can hardcode the word status, because it needs to be able to be translated programmatically, but it could use the word status by default.